src/main/webapp/resources/static/css/ootb/page-templates/index.css

```diff
@@ -0,0 +1,138 @@
+/* http://meyerweb.com/eric/tools/css/reset/
+   v5.0.1 | 20191019
+   License: none (public domain)
+*/
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, menu, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+main, menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  vertical-align: baseline; }
+
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, main, menu, nav, section {
+  display: block; }
+
+/* HTML5 hidden-attribute fix for newer browsers */
+*[hidden] {
+  display: none; }
+
+body {
+  line-height: 1; }
+
+menu, ol, ul {
+  list-style: none; }
+
+blockquote, q {
+  quotes: none; }
+
+blockquote:before, blockquote:after,
+q:before, q:after {
+  content: '';
+  content: none; }
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0; }
+
+
+:root {
+  --primary-color: #002F87;
+  --primary-background-color: #FFFFFF;
+  --secondary-background-color: #F4F4F8;
+  --nav-link-text-color: #C6C6C6;
+  --primary-button-background-color: #1062FE;
+  --default-button-background-color: #E5E5E5;
+}
+
+.Homepage1__header {
```

Comment:
For this one and other `HomepageX__header` classes, try to extract the common part to a `Homepage__header` class, then use the specific parts using `.HomepageX.Homepage__header` selector.

Also ensure all is working fine, because the HTML templates only had `HomepageX_header` (with only one underscore char instead of two).